forge-game/src/main/java/forge/game/player/Player.java

```diff
@@ -543,6 +543,10 @@ public final boolean canGainLife() {
     }
 
     public final int loseLife(int toLose, final boolean damage, final boolean manaBurn) {
+        return loseLife(toLose, damage, manaBurn, false);
```

Comment:
is this extra param really needed / useful?
Seems to me the logic is when `damage = true` GameAction needs to do it

Reply:
unless Cost with Player losing Life